### PR TITLE
Update Istio stable to 1.7.1 and latest to 1.7.3

### LINKED ISSUE #312: Move stable to Istio 1.7
As we see the performance regression of Istio 1.5, we should move the stable to Istio 1.7.

### PR DESCRIPTION
**Pull request by arturenault**
Fixes #312 

- Update istio stable to 1.7 to fix performance issues
- Update istio latest to the latest release

### CODE REVIEW

**Comment by JRBANCEL** [comment]
It is not that simple 😄 
`istio-latest` uses the Operator model, while `stable` uses the Helm model.
In 1.7.x, both should be using the Operator model.
So, copy `istio-latest` to `istio-stable` and update the versions.

File: third_party/istio-latest/install-istio.sh [hunk 1]
```diff
@@ -24,7 +24,7 @@ if [ $? != 0 ]; then
   echo "Failed to download Istio package"
   exit 1
 fi
-tar xzf ${ISTIO_TARBALL}
```
Comment (JRBANCEL) [inline third_party/istio-latest/install-istio.sh:27]: ?

**Comment by JRBANCEL** [comment]
Can you use a more descriptive title like: `Stable = 1.7.1, Latest = 1.7.3`?

**Review by tcnghia — APPROVED** [review]
/lgtm

**Comment by tcnghia** [comment]
(for @JRBANCEL review)

**Pull request merged**